apps/dashboard/src/routes/lms/community/[slug]/+page.svelte

```diff
@@ -227,7 +241,11 @@
 
   async function handleQuestionEdit() {
     if (isEditMode) {
-      const fields = { title: editContent.title, body: editContent.body };
+      const fields = {
+        title: editContent.title,
+        body: editContent.body,
+        course_id: editContent.course_id
```

Comment:
@tunny17 I keep telling you to change this to camel case, the 3rd time.

bro what's up??? I am just wasting my time everytime on these PR reviews.